termwiz/src/color.rs

```diff
@@ -71,31 +69,34 @@ impl RgbColor {
         Self { red, green, blue }
     }
 
-    pub fn to_linear(self) -> LinSrgba {
-        Srgba::<u8>::new(self.red, self.green, self.blue, 0xff)
-            .into_format()
-            .into_linear()
-    }
-
     pub fn to_tuple_rgba(self) -> RgbaTuple {
-        Srgba::<u8>::new(self.red, self.green, self.blue, 0xff)
-            .into_format()
-            .into_components()
-    }
-
-    pub fn to_linear_tuple_rgba(self) -> RgbaTuple {
-        self.to_linear().into_components()
+        (
```

Comment:
This is the one thing I'm unsure about in this PR; there are some subtle differences between SRGB and RGB to do with the non-linear scaling in SRGB.  I think this change is good, but it's worth making sure that the old implementation was just doing simple divide by 255; the consequence of this not being right is that the colors will have the wrong brightness.  This is more obvious with brighter color schemes!

Reply:
I kept both functions, although this to_linear one does not seem to be used.